main.nf

```diff
@@ -933,23 +1366,115 @@ process qualimap {
     script:
     gcref = params.genome.toString().startsWith('GRCh') ? '-gd HUMAN' : ''
     gcref = params.genome.toString().startsWith('GRCm') ? '-gd MOUSE' : ''
-    def avail_mem = task.memory ? ((task.memory.toGiga() - 6) / task.cpus).trunc() : false
-    def sort_mem = avail_mem && avail_mem > 2 ? "-m ${avail_mem}G" : ''
-    """
-    samtools sort $bam \\
-        -@ ${task.cpus} $sort_mem \\
-        -o ${bam.baseName}.sorted.bam
-    qualimap bamqc $gcref \\
-        -bam ${bam.baseName}.sorted.bam \\
-        -outdir ${bam.baseName}_qualimap \\
-        --collect-overlap-pairs \\
-        --java-mem-size=${task.memory.toGiga()}G \\
-        -nt ${task.cpus}
-    """
+	"""
+	qualimap bamqc $gcref \\
+		-bam ${bam.baseName}.bam \\
+		-outdir ${bam.baseName}_qualimap \\
+		--collect-overlap-pairs \\
+		--java-mem-size=${task.memory.toGiga()}G \\
+		-nt ${task.cpus}
+
+	"""
 }
 
+
+ /*
+ * STEP 9 - Picard - Preparation step
+ */
+  process prepareGenomeToPicard {
+	publishDir path: { params.save_reference ? "${params.outdir}/reference_genome" : params.outdir },
+                   saveAs: { (params.save_reference && it.indexOf("dict") >0) ? it : null }, mode: 'copy' 
+
+      input:
+      file fasta from ch_fasta_for_picard
+	file file_try from ch_try
```

Comment:
what is this used for?

Reply:
I wanted to make sure that "get_software_versions" run first in the list of processes, because I had some problems with that... Sorry, my mistake I forgot to remove that. Fixed